### PR TITLE
LPS-197484 | Test Fix master

### PR DESCRIPTION
**Pull request by leilanyaragao**
Jira Issue: https://liferay.atlassian.net/browse/LPS-197484

### CODE REVIEW

**Comment by leilanyaragao** [comment]
ci:test:sf

**Comment by leilanyaragao** [comment]
ci:test:relevant

**Comment by leilanyaragao** [comment]
ci:test:object

**Comment by leilanyaragao** [comment]
ci:test:object

File: modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountEntryLocalServiceTest.java [hunk 1]
```diff
@@ -199,13 +200,25 @@ public void testAccountEntryObjectValidationRule() throws Exception {
...
+					);
+
+			String errorMessage = objectValidationRuleResult.getErrorMessage();
+
+			Assert.assertTrue(errorMessage.contains("This name is invalid."));
 
 			Assert.assertTrue(
 				modelListenerException.getCause() instanceof
-					ObjectValidationRuleEngineException.InvalidFields);
+					ObjectValidationRuleEngineException);
```
Comment (marcelabc) [inline modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountEntryLocalServiceTest.java:221]: Can we move this before the variable creation `objectValidationRuleEngineException` ?

File: modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountEntryLocalServiceTest.java [hunk 2]
```diff
@@ -215,13 +228,25 @@ public void testAccountEntryObjectValidationRule() throws Exception {
...
+						modelListenerException.getCause();
+
+			ObjectValidationRuleResult objectValidationRuleResult =
+				objectValidationRuleEngineException.
+					getObjectValidationRuleResults(
+					).get(
+						0
+					);
+
+			String errorMessage = objectValidationRuleResult.getErrorMessage();
```
Comment (marcelabc) [inline modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountEntryLocalServiceTest.java:243]: Same considerations from previous comment :+1:

**Comment by leilanyaragao** [comment]
ci:test:sf

**Comment by leilanyaragao** [comment]
ci:test:object

**Comment by leilanyaragao** [comment]
ci:test:relevant

**Comment by leilanyaragao** [comment]
ci:test:sf

**Comment by leilanyaragao** [comment]
ci:test:relevant

**Comment by leilanyaragao** [comment]
ci:test:object

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**